File: CSX/Functions and Execution Context/multiplyAll.js
function multiplyAll(...args) {
  let total = 1;
  for (let arg of args) {
    total *= arg;
  }

  return total;
}

console.log(multiplyAll(9, 4, 5, 6, 7, 2, 1, 8, 3)) 
console.log(multiplyAll(5, 5, 5, 3)) 

File: CSX - Python/Functions and Execution Context/multiply_all.py
def multiply_all(*nums):
    total = 1
    for num in nums:
        total *= num
    return total

print(multiply_all(9, 4, 5, 6, 7, 2, 1, 8, 3))
print(multiply_all(5, 5, 5, 3))
